corehq/apps/linked_domain/templates/linked_domain/tabs/push_release_content_tab.html

```diff
@@ -36,9 +36,7 @@ <h3>{% trans "Push Content" %}</h3>
         </label>
         <div class="col-sm-9 col-md-10 controls">
           <select multiple class="form-control"
-                  data-bind="selectedOptions: domainsToPush,
-                             multiselect: multiselectProperties,
-                             multiselectOptions: localDownstreamDomains">
+                  data-bind="selectedOptions: domainsToPush, multiselectProperties: multiselectProperties">
```

Comment:
does `selectedOptions` work with the multiselect? I had some difficulty with it.

Reply:
Yeah in my experience I haven't had any issues with the `selectedOptions`, but definitely have issues with the `options` binding. And the linked domain code relied on that working prior to any recent changes in this area. For example, here:
https://github.com/dimagi/commcare-hq/blob/daaa9f51b5b9de1c5df2a90d54f2881b0eff6b55/corehq/apps/linked_domain/templates/linked_domain/tabs/push_release_content_tab.html#L21-L26